src/commonMain/kotlin/fr/acinq/lightning/NodeParams.kt

```diff
@@ -154,7 +154,8 @@ data class NodeParams(
     val liquidityPolicy: MutableStateFlow<LiquidityPolicy>,
     val minFinalCltvExpiryDelta: CltvExpiryDelta,
     val maxFinalCltvExpiryDelta: CltvExpiryDelta,
-    val bolt12invoiceExpiry: Duration
+    val bolt12invoiceExpiry: Duration,
+    val maxPayerNoteLength: Int,
```

Comment:
I'm not sure it makes sense to let wallets pick a value here. Shouldn't we define a maximum value, expose it to the wallet and document it, and reject `PayOffer` that set a large payer note? Then wallets know that they should limit the textbox they display to users.

Reply:
I've hardcoded 64 characters as the limit. Wallets should probably limit the textbox, however I think that's purely a UI consideration and we shouldn't reject valid payment attempts.

Reply:
Good point, it's the responsibility of the wallet to decide the number of characters to allow :+1: